### DISCUSSION TITLE
vn.train returns False

### DISCUSSION THREAD

**Discussion by LanaLapitskaya** [vanna-ai/vanna#216] [Q&A]
Hello, I've tried to train the model like it is described in docs https://vanna.ai/docs/snowflake-openai-vanna-vannadb.html, but vn.train function returned False for all attempts to train using ddl, sql and documentation. When I tried to train using database information schema, it also returned False and the message "Not able to add documentation for [name of my table]". Could you please help me to solve this?

**Comment by zainhoda** [comment]
The most likely thing that's happening is you're trying to train a model that you don't own. Ensure that the model you're using is listed here under "Your RAG Models"
https://vanna.ai/account/profile

Note that the model name is always lowercase.

For additional help, please join the Discord. It's usually easier to help debug live
